### DISCUSSION TITLE
how would i use things like buycraft/tebex with a minestom server

### DISCUSSION THREAD

**Discussion by agwoodev** [Minestom/Minestom#2308] [Help]
because there is literaly nothing on minestom by defult how would i use things like luckperms and tebex

**Comment by JahsehJ** [comment]
you need to implement the api yourself. here are the official docs -> https://docs.tebex.io/developers
hollowcube has done this but i don't think it's open source.

**Reply by mudkipdev** [reply]
Hollow Cube uses Go for Tebex, not Java.

**Reply by JahsehJ** [reply]
mfw microservices

**Comment by Nopock** [comment]
iirc someone implemented tebex awhile ago, it would likely need an update but should be pretty easy to do. and loofifteen has a luckperms fork for minestom: https://github.com/LooFifteen/LuckPerms
